server/models/blog.js

```diff
@@ -0,0 +1,22 @@
+import mongoose, { Schema } from 'mongoose';
+
+const blogsSchema = new Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  intro: {
+    type: String,
+    required: true,
+  },
+  content: {
+    type: String,
+    required: true,
+  },
+  date: {
+    type: String,
+    required: true,
+  },
+});
+
+module.exports = mongoose.model('Blog', blogsSchema);
```

Comment:
This file too. It shouldn't apprear here. The only changes that should be here are those that deal with adding a single request